src/main/java/com/monumental/models/VerificationToken.java

```diff
@@ -0,0 +1,59 @@
+package com.monumental.models;
+
+import com.sun.istack.NotNull;
```

Comment:
I don't think this is the `@NotNull` we're using everywhere else.

Reply:
Good catch